test/index.test.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const Bluebird = require( '../lambda-src/node_modules/bluebird')
+var expect = require( '../lambda-src/node_modules/chai' ).expect;
+
+var myLambda = require( '../lambda-src/index' );
+
+
+const request = require('../lambda-src/node_modules/request-promise-native');
+
+const sinon = require('../lambda-src/node_modules/sinon')
+
+
+
+const WEBHOOK_URL = 'http://localhost/test'
+let event_message = require( './aws_budgtets_test_message.json' );
+// let sandbox = sinon.createSandbox()
+
+// console.log(event_message)
```

Comment:
These commented lines should be removed.

Reply:
Removed